app/helpers/agent_helper.rb

```diff
@@ -82,6 +82,12 @@ def agent_type_icon(agent, agents)
     end
   end
 
+  def agent_type_select_options
+    Rails.cache.fetch('agent_type_select_options') do
+      Agent.types.map {|type| [agent_type_to_human(type.name), type, {title: h(Agent.build_for_type(type.name, User.new(id: 0), {}).html_description.lines.first.strip)}] }
```

Comment:
There are a couple agents that show things like agent.user_id in there to give you example URLs. We could factor these out, but for now it's not really class level.